pkgs/by-name/fl/flatito/package.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, fetchFromGitHub, ruby, buildRubyGem, bundlerEnv }:
+let
+  deps = bundlerEnv rec {
+    name = "flatito-${version}";
+    source.sha256 = lib.fakeSha256;
```

Comment:
Is this normal ?

Reply:
It seems it can be removed the source.sha256 thing. I don't know why is was there :)